test/cluster.sh

```diff
@@ -61,11 +61,13 @@ function create_monitoring() {
 }
 
 function create_everything() {
-  create_istio
-  create_serving
-  create_test_resources
+  if [[ -z $USING_EXISTING_SERVING ]]; then
```

Comment:
You can run the tests against an existing Knative instance using `go test` and previously uploading the test images, as explained in the README. Although I understand the motivation, I would prefer an explicit flag for running the test rather than an obscure env var that changes the inner behavior of a helper function.

Reply:
OK. Makes sense